lib/gui/index.html

```diff
@@ -49,5 +50,15 @@
         manifest-bind="version"
         os-open-external="https://github.com/resin-io/etcher/blob/master/CHANGELOG.md"></span>
     </footer>
+
+    <div class="section-loader"
+      ng-controller="StateController as state"
+      ng-class="{
+        isFinish: state.is('success')
+      }">
+      <safe-webview
+        src="'https://etcher.io/success-banner/'"
```

Comment:
Actually, this value is not supposed to change on the fly, so lets make it an interpolation @Shou